sources/web/datalab/config/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Running with --live, the container creates a metadata file here.
+metadata.json
```

Comment:
I think this file is obsolete anyway, I can't see where we use it. This is separate from this change though.